sdk/communication/azure-communication-sms/src/test/resources/session-records/SmsAsyncClientTests.sendSmsToSingleNumberWithOptions[1].json

```diff
@@ -3,19 +3,19 @@
     "Method" : "POST",
     "Uri" : "https://REDACTED.communication.azure.com/sms?api-version=2021-03-07",
     "Headers" : {
-      "User-Agent" : "azsdk-java-azure-communication-sms/1.0.0-beta.5 (15.0.1; Windows 10; 10.0)",
+      "User-Agent" : "azsdk-java-azure-communication-sms/1.0.0-beta.5 (11.0.8; Windows 10; 10.0)",
       "Content-Type" : "application/json"
     },
     "Response" : {
       "Transfer-Encoding" : "chunked",
       "api-supported-versions" : "2020-07-20-preview1, 2020-08-20-preview, 2021-03-07",
-      "X-Processing-Time" : "233ms",
-      "MS-CV" : "xN3QN5dSFU+uXAh3tgfkwQ.0",
+      "X-Processing-Time" : "620ms",
+      "MS-CV" : "kNlN2GFqUU2EY8PANxGa9g.0",
       "retry-after" : "0",
-      "X-Azure-Ref" : "0JhtAYAAAAABfsATFLhVkR74NjHRlyqm+REFMRURHRTEwMTMAOWZjN2I1MTktYThjYy00Zjg5LTkzNWUtYzkxNDhhZTA5ZTgx",
+      "X-Azure-Ref" : "0Fn5CYAAAAAC0dNZhMPh0Rqimy8F9GyTpWVZSMzBFREdFMDQwOAA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
       "StatusCode" : "202",
-      "Body" : "{\"value\":[{\"to\":\"REDACTED\",\"messageId\":\"Outgoing_20210303232630ca1305d7-1a32-404d-ba19-387d7b9aae1a_noam\",\"httpStatusCode\":202,\"successful\":true}]}",
-      "Date" : "Wed, 03 Mar 2021 23:26:30 GMT",
+      "Body" : "{\"value\":[{\"to\":\"REDACTED\",\"messageId\":\"Outgoing_2021030518531146333cd2-4143-40d5-903f-b5b87b63756b_noam\",\"httpStatusCode\":202,\"successful\":true}]}",
```

Comment:
We're sanitizing messageId in TS and Python. If this doesn't break your test I'd suggest doing it.
(I've changed my compare 2 IDs test to only run Live to get around this for instance)

Reply:
Hey Bert, we should only be sanitizing tokens and PII. Sanitizing message Id might be over-sanitizing.